mono/utils/mono-proclib.c

```diff
@@ -826,13 +826,13 @@ get_cpu_times (int cpu_id, gint64 *user, gint64 *systemt, gint64 *irq, gint64 *s
 {
 	char buf [256];
 	char *s;
-	int hz = get_user_hz ();
+	int uhz = get_user_hz ();
```

Comment:
Is this a name clash? Is hz taken?

Reply:
Yes, AIX headers define hz as a macro with the compiler flags we use.